api/numaresourcesoperator/v1/helper/nodegroup/nodegroup_test.go

```diff
@@ -420,3 +461,51 @@ func findListByNodeGroups(mcps *mcov1.MachineConfigPoolList, nodeGroups []nropv1
 
 	return result, nil
 }
+
+func TestContainsSliceItems(t *testing.T) {
```

Comment:
if you're using `set.Set` you don't need the `containsSliceItems` function and this unit test either